src/main/kotlin/io/github/corbym/dokker/dokker.kt

```diff
@@ -9,9 +10,45 @@ fun dokker(init: DokkerContainerBuilder.() -> Unit): DokkerContainer {
     return DokkerContainerBuilder().apply(init).build()
 }
 
+// Search order is
+// 1. process name set in code (if set, DokkerAutoProcessSearchResult.processName is not used)
+// 2. environment variable (if found in path)
+// 3. hardcoded "docker" (if found in path)
+// 3. hardcoded "podman" (if found in path)
+internal object DokkerAutoProcessSearchResult {
+
+    // Note that "command -v" - this is best-effort as adding an 'alias' will cause issues
+    // MacOS defaults to zsh shell.
+    //   Intellij does not read ~/.zprofile nor ~/.zshrc scripts so set environment variables in ~/.profile
+    //   Otherwise, "command -v" may not find the executable
+    private fun processFullPath(process: String): String? {
+        return try {
+            "command -v $process".runCommand()
+        } catch (e: Exception) {
```

Comment:
Can we remove the exception - I'd rather have this throw for SecurityExceptions as I think it's something we should be telling the user